volumetric/homotopicThinning3D.cpp

```diff
@@ -143,8 +147,13 @@ int main( int argc, char** argv )
       trace.error()<< " The coordinates should be 3d coordinates, ignoring fixedPoints option." << std::endl;
     }
   }
-
-
+  if(vm.count("fixedPointSDP")){
+    std::vector<Z3i::Point> vPt = PointListReader<Z3i::Point>::getPointsFromFile(vm["fixedPointSDP"].as<std::string>());
+    for( auto &p: vPt){
```

Comment:
please check you IDE to match with DGtal parenthesis/indent conventions.  ```{``` in new lines for instance.